tests/bfd/test_bfd.py

```diff
@@ -40,13 +41,17 @@ def get_t0_intfs(mg_facts):
 
 
 def add_dut_ip(duthost, intfs, ips, prefix_len):
+    buffer =""
     for idx in range(len(intfs)):
-        duthost.shell('sudo config interface ip add {} {}/{}'.format(intfs[idx], ips[idx], prefix_len))
+        buffer += 'sudo config interface ip add {} {}/{} ;'.format(intfs[idx], ips[idx], prefix_len)
```

Comment:
suggest use name as "cmd" since buffer is ambiguous.

Reply:
done.